src/Spec2-Adapters-Morphic/SpMorphicWorldAdapter.class.st

```diff
@@ -19,9 +19,26 @@ SpMorphicWorldAdapter >> addPresenterIn: widgetToBuild withSpecLayout: aSpec [
 SpMorphicWorldAdapter >> buildWidget [
 
 	^ self presenter currentWorld 
+		borderColor: self theme backgroundColor;
+		borderWidth: 4;
```

Comment:
this should be done through a style, I think?

Reply:
still, I can accept it because this is too morphic specific. Tell me if it is too hard.

Reply:
yes, really morphic specific. Also, I do not see any use of border in the style nor how to use colors from the theme. Let's go fast for this time. We will review that with gtk backend